frontend/games/src/App.vue

```diff
@@ -0,0 +1,21 @@
+<template>
+<div class="tile is-ancestor">
+  <div class="tile is-6 is-vertical is-parent">
+    <div class="tile is-child box has-text-centered">
+      <button class="button is-primary">Add Game</button>
+    </div>
+    <div class="tile is-child box">
+      <h3 class="title">Player stats</h3>
+    </div>
+  </div>
+  <div class="tile is-parent">
+    <div class="tile is-child box">
+      <h3 class="title">Recent games</h3>
+    </div>
+  </div>
+</div>
+</template>
+
+
+<script>
+</script>
```

Comment:
Best not to be committing binary files like `.ico` and `png` - consider hosting them somewhere (like imgur). Security reasons mostly